examples/fluid-solid-bathymetry/.gitignore

```diff
@@ -0,0 +1,3 @@
+line_sources.yaml
```

Comment:
Does this make sense? 
Do you not want line_sources to be tracked?

Reply:
Yup, for now I do. Thanks!

I am planning to add `jinja2` [templating](https://realpython.com/primer-on-jinja-templating/) for generating the sources programmatically later. Then I wouldn't want that file to be tracked, but the templating file to be tracked